frontend/src/components/common/Button/Button.tsx

```diff
@@ -3,10 +3,11 @@ import styled, { css } from 'styled-components';
 
 import type { ButtonHTMLAttributes, MouseEventHandler } from 'react';
 
-type DirectionType = 'all' | 'top' | 'bottom';
+import type { DirectionType } from 'types/style';
```

Comment:
```suggestion
import type { BorderRadiusDirectionType } from 'types/style';
```
이렇게 하는게 더 좋을거 같습니다~~